_opensearch/search/sort.md

```diff
@@ -0,0 +1,880 @@
+        "sort" : [
+          3200
+        ]
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "3198",
+        "_score" : null,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 3199,
+          "play_name" : "Henry IV",
+          "speech_number" : 8,
+          "line_number" : "5.5.40",
+          "speaker" : "KING HENRY IV",
+          "text_entry" : "Myself and you, son Harry, will towards Wales,"
+        },
+        "sort" : [
+          3199
+        ]
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "3197",
+        "_score" : null,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 3198,
+          "play_name" : "Henry IV",
+          "speech_number" : 8,
+          "line_number" : "5.5.39",
+          "speaker" : "KING HENRY IV",
+          "text_entry" : "Who, as we hear, are busily in arms:"
+        },
+        "sort" : [
+          3198
+        ]
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "3196",
+        "_score" : null,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 3197,
+          "play_name" : "Henry IV",
+          "speech_number" : 8,
+          "line_number" : "5.5.38",
+          "speaker" : "KING HENRY IV",
+          "text_entry" : "To meet Northumberland and the prelate Scroop,"
+        },
+        "sort" : [
+          3197
+        ]
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "3195",
+        "_score" : null,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 3196,
+          "play_name" : "Henry IV",
+          "speech_number" : 8,
+          "line_number" : "5.5.37",
+          "speaker" : "KING HENRY IV",
+          "text_entry" : "Towards York shall bend you with your dearest speed,"
+        },
+        "sort" : [
+          3196
+        ]
+      }
+    ]
+  }
+}
+```
+
+The `sort` parameter is an array, so you can specify multiple field values in the order of their priority.
+
+If you have two fields with the same value for `line_id`, OpenSearch uses `speech_number`, which is the second option for sorting.
+
+```json
+GET shakespeare/_search
+{
+  "query": {
+    "term": {
+      "play_name": {
+        "value": "Henry IV"
+      }
+    }
+  },
+  "sort": [
+    {
+      "line_id": {
+        "order": "desc"
+      }
+    },
+    {
+      "speech_number": {
+        "order": "desc"
+      }
+    }
+  ]
+}
+```
+
+You can continue to sort by any number of field values to get the results in just the right order. It doesn’t have to be a numerical value---you can also sort by date or timestamp fields:
+
+```json
+"sort": [
+    {
+      "date": {
+        "order": "desc"
+      }
+    }
+  ]
+```
+
+A text field that’s analyzed cannot be used to sort documents, because the inverted index only contains the individual tokenized terms and not the entire string. So, you cannot sort by the `play_name`, for example.
```

Comment:
```suggestion
A text field that is analyzed cannot be used to sort documents, because the inverted index only contains the individual tokenized terms and not the entire string. So you cannot sort by the `play_name`, for example.
```